CommunityProject/Assets/_Game/Scripts/Editor/StripDevelopmentOnlyObjectsFromBuild.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace BoundfoxStudios.CommunityProject.Editor
+{
+	public class StripDevelopmentOnlyObjectsFromBuild : IProcessSceneWithReport
+	{
+		public int callbackOrder { get; }
+		public void OnProcessScene(Scene scene, BuildReport report)
```

Comment:
```suggestion
		public void OnProcessScene(Scene scene, BuildReport? report)
```

Mit den nullable reference types wäre `report is null` immer false, deswegen markieren das wir hier auch null erwarten.

Reply:
@M4lph Guter Fang, hatte ich gesehen, aber nicht beachtet, weil ich dachte, dass dann das Interface nicht mehr erfüllt wird. Gerade ausprobiert und das Interface zählt weiterhin als erfüllt. Das ist gut :)